src/tests/plugin.test.ts

```diff
@@ -178,27 +178,23 @@ describe("tests for sql generation by plugins", () => {
             ...options,
             values: [],
             keywordsLowerCase: true,
-          })
+          }),
         ).toBe("table.column = any($1)");
       });
     });
 
     describe("invariant", () => {
       it("should fail on falsy", () => {
         expect(() =>
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/ban-types
-          (MapInToEqualsAnyPlugin.invariant as Function)({
-            operands: null,
-          })
+          MapInToEqualsAnyPlugin.invariant({} as unknown as ComparisonNode),
```

Comment:
remove the suppressions by changing the declaration of the plugin and using a cast.